tests/mail/file.txt

```diff
@@ -0,0 +1 @@
+django/django
```

Comment:
It would be nice to put these in an "attachments" directory to avoid cluttering the base directory.